ckanext/budgetviz/groupbarchart_munc/theme/public/custom.css

```diff
@@ -15,6 +15,9 @@
 }
 
 .chartarea {
-    margin-top: 20px;
+    padding: 10px 25px;
```

Comment:
The visualizations arent mobile friendly, because of the structure. Considering the purpose of exploring data sets on data portal doesnt serve well on small screen devices, making visualizations mobile friendly has been low on priority.